src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: solid 1px $card-border-color;
+  box-sizing: border-box;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+  }
+
+  &__text {
+    color: $card-main-accent-color;
+    width: 166px;
+    margin: 0 auto;
+    margin-top: 40px;
+    font-weight: $font-weight5;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    color: $card-secondary-color;
+    font-weight: $font-weight4;
```

Comment:
you use this font the most times, let's make it the default font?

Reply:
I expect to use other font then as it is only a card which doesnt have a huge impact on other website elements. Please accept my work as I did the whole job and explained why I did such thing before